Sources/Bow/Data/Try.swift

```diff
@@ -223,6 +216,14 @@ extension TryPartial: EquatableK {
     }
 }
 
+// MARK: Instance of HashableK for Try
+extension TryPartial: HashableK {
+    public static func hash<A>(_ fa: TryOf<A>, into hasher: inout Hasher) where A : Hashable {
+        fa^.value.foldRun { hasher.combine("\($0)") }
+                       _: { hasher.combine($0) }
```

Comment:
Please, use the parenthesis versions to try to maintain compatibility with previous Swift versions.